app/assets/stylesheets/application.scss

```diff
@@ -16,6 +16,7 @@
 @import 'spotlight/spotlight';
 
 @import "components/universal_viewer";
+@import "components/masthead";
```

Comment:
What constitutes a "component"? Since we have both View Components and Vue Components maybe we should be more specific about language?

Reply:
(Not a blocking concern.)

Reply:
Oh I have no idea. 😂  I'd love to know where I should put this. What do you think?

Reply:
Or "partials"? Would that make sense?